### DISCUSSION TITLE
How to Change the Background of Silder when pointer over or pointer pressed?

### DISCUSSION THREAD

**Discussion by haijialiu** [microsoft/microsoft-ui-xaml#10072] [Q&A] [answered]
I'm a noob, I set the Background of Silder, but it just change the normal status, when you pointer over it,  the Background changes back. It's any simple method can set the background in all status?

**Comment by castorix** [comment, marked as answer]
(it is Slider, not Silder...)
You can change the resources
For example :
```
Application.Current.Resources["SliderTrackFill"] = new SolidColorBrush(Microsoft.UI.Colors.RoyalBlue);
Application.Current.Resources["SliderTrackFillPointerOver"] = new SolidColorBrush(Microsoft.UI.Colors.RoyalBlue);
Application.Current.Resources["SliderTrackFillPressed"] = new SolidColorBrush(Microsoft.UI.Colors.RoyalBlue);
```

**Reply by haijialiu** [reply]
Thanks a lot(yeah,Slider is not silder),it's very useful for me, have a good day. :)

**Reply by NicholasChrzan** [reply]
The above will replace the resources globally. If you want to replace for a single instance then you can do the same by specifying a resource with the same name.
i.e.

```
<Slider.Resources>
<ResourceDictionary>
<SolidColorBrush x:Key="SliderTrackFill" Color="RoyalBlue"/>
<SolidColorBrush x:Key="SliderTrackFillPointerOver" Color="RoyalBlue"/>
<SolidColorBrush x:Key="SliderTrackFillPressed" Color="RoyalBlue"/>
</ResourceDictionary>
</Slider.Resources>
```
